source.js

```diff
@@ -10,7 +10,7 @@ function template(moduleName, options) {
     options = {};
   }
   var str = templateSTR.replace(/defineNamespace\(\)/g, compileNamespace(moduleName))
-    .split('source()')
+      .replace(/deps\(\)/g, getDependencies(options.deps)).split('source()')
   str[0] = str[0].trim();
```

Comment:
I think it could be clearer formatted like this:
```js
   var str = templateSTR
    .replace(/defineNamespace\(\)/g, compileNamespace(moduleName))
    .replace(/deps\(\)/g, getDependencies(options.deps))
    .split('source()')
```